### ISSUE TITLE
Issue with GUI position, and other mods that add other tabs .

### ISSUE THREAD

**Issue by crunkazcanbe** [Funwayguy/InfiniteInvo#39]
The only mods installed is Tinkers Construct ,and Galacticraft . Look at the images http://imgur.com/ouw07SO,S0JIeKH,8fkm4nk There is a image of each tab being clicked on .

This is my config for InfiniteInvo here http://pastebin.com/fEpsT1we

**Comment by Funwayguy** [comment]
There seems to be an error of only a few pixels to the left of where it should be. It doesn't look like a serious issue but I will see if there are any calculation errors that need to be corrected

**Comment by Funwayguy** [comment]
Oh I just noticed the galacticraft tab... yea that definitely needs to be fixed too

**Comment by crunkazcanbe** [comment]
Thanx man. Sorry for the issue . Just thought I would get this out there for you .

**Comment by Funwayguy** [comment]
Update: I'm not 100% sure but I might have fixed this. I'd need to double check with Galacticraft installed to be sure

**Comment by Funwayguy** [comment]
Nope, still broken. Galacticraft is doing something similar to InfiniteInvo with extending the vanilla inventory and the two are being confused about who controls which slots.
